data-es6.js

```diff
@@ -8768,15 +8768,16 @@ exports.tests = [
         var get = [];
         var p = new Proxy({ exec: function() { return null; } }, { get: function(o, k) { get.push(k); return o[k]; }});
         RegExp.prototype[Symbol.search].call(p);
-        return get + '' === "lastIndex,exec";
+        return get + '' === "lastIndex,exec,lastIndex";
       */},
+      note_id: "regexp-prototype-symbolsearch",
```

Comment:
I believe we do this when the note is about the result.  There several instances of notes on top-level ("block-level function declaration", "well-known symbols", "non-strict function semantics", etc).  The trouble is I found no other note associated with a subtest instead of the main test, and that's why I believe there's a bug (or limitation) on the generator.